### ISSUE TITLE
Android screen locking should not stop app

### ISSUE THREAD

**Issue by oadams** [langtech/aikuma#31]
Stop the respeaking activity when the phone is locked (however if the screen goes blank from a timeout, continue the respeaking).

**Comment by stevenbird** [comment]
@Lingomat would you please clarify the issue? When you write "When Android's inactivity timeout blanks the screen, respeaking stops", what do you mean by "respeaking stops"?

I think we want to make sure that recording an original source does not stop. I verified that display sleep does not happen during an original recording.

**Comment by Lingomat** [comment]
It used to be display sleep, while recording in respeaking would stop the respeaking recording. However this was in the voice activated respeaking mode. We've since moved to thumb-based commentary and the phone will note display sleep while you're holding record (tested).

I took a quick fresh look at it. On the Marketplace version of Aikuma2, when I switched it to voice activated I couldn't get it to play the original at all. So I can't test if the problem persists.
